css/styles.css

```
@@ -0,0 +1,93 @@
+/*
+- Font sizes (px):
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+
+- Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+*/
+
+:root {
+  font: 400 10px / 1.2 "Yeseva One";
+}
+
+.container {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  height: 12rem;
+  gap: 0.4rem;
+  padding-left: 18rem;
+  padding-right: 20.4rem;
+  align-items: center;
+}
+
+.logo {
+  font-size: 3.6rem;
+  text-transform: uppercase;
+  margin-right: auto;
+}
+
+.nav {
+  font-family: "Gentium Book Plus";
+  font-weight: bold;
+  align-self: stretch;
+}
+
+.nav-list {
+  display: flex;
+  gap: 2.2rem;
+  height: 100%;
+  align-items: center;
+}
+
+.nav-item {
+  position: relative;
+  display: flex;
+  align-self: stretch;
+}
+
+.nav-link {
+  font-size: 2.1rem;
+  padding-left: 1.4rem;
+  padding-right: 1.4rem;
+  align-self: center;
+}
+
+.nav-link.nav-link-active .nav-link-text::before,
+.nav-link:not(.nav-link-active) .nav-link-text:hover::before {
+  content: "";
+  width: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-top: 4px solid #c8c8c8;
+}
```

Consider this solution:
```suggestion
.nav-link::before {
  opacity: 0;
  content: "";
  width: 100%;
  position: absolute;
  top: 0;
  left: 0;
  height: 3px;
  background: #fff;
  transition: opacity 0.3s ease-in;
}

.nav-link.nav-link-active::before, 
.nav-link:hover::before {
  opacity: 0.8;
}
```
So every nav-link element has this line, but it only visible on hover or when is active.